src/libultra_code/libultra_internal.h

```diff
@@ -43,4 +43,6 @@ s32 __osAiDeviceBusy();
 void __osDispatchThread();
 u32 __osGetCause();
 s32 __osAtomicDec(u32*);
+signed short sins(unsigned short x);
+signed short coss(unsigned short x);
```

Comment:
I don't think these prototypes belong here, but either way they should use the proper ultra types.
```suggestion
s16 sins(u16 x);
s16 coss(u16 x);
```